packages/blitz-auth/build.config.ts

```diff
@@ -2,7 +2,7 @@ import {BuildConfig} from "unbuild"
 
 const config: BuildConfig = {
   entries: ["./src/index-browser", "./src/index-server"],
-  externals: ["index-browser.cjs", "index-browser.mjs", "react"],
+  externals: ["index-browser.cjs", "index-browser.mjs", "react", "next-auth", "next-auth/jwt"],
```

Comment:
add as an optional peer dependency instead of here, otherwise pnpm may not work right

Reply:
okay I will do that

Reply:
still todo